src/base/ghost_point_neighbors.C

```diff
@@ -36,6 +42,23 @@ void GhostPointNeighbors::operator()
 {
   libmesh_assert(_mesh);
 
+#ifdef LIBMESH_ENABLE_PERIODIC
+  bool check_periodic_bcs =
+    (_periodic_bcs && !_periodic_bcs->empty());
+
+  std::unique_ptr<PointLocatorBase> point_locator;
+  if (check_periodic_bcs)
+    {
+      libmesh_assert(_mesh);
```

Comment:
Worried the mesh might have disappeared in the last 9 lines?

Reply:
Uh yea! You can never be too sure. Haha I will fix that